v4/export/writer_util.go

```diff
@@ -36,67 +94,103 @@ func WriteInsert(tblIR TableDataIR, w io.StringWriter) error {
 		return nil
 	}
 
+	pool := sync.Pool{New: func() interface{} {
+		return &bytes.Buffer{}
+	}}
+	bf := pool.Get().(*bytes.Buffer)
```

Comment:
What's the point of the pool? `pool.Get()` is only called in line 100 so nothing is being reused.

Reply:
I have put pool has a global variable. If `WriteInsert` is called for other tables I think is can be reused.